build.gradle.kts

```diff
@@ -316,7 +316,7 @@ subprojects {
     }
 }
 
-val checkCopyright by tasks.registering(Exec::class) {
+tasks.register<Exec>("checkCopyright") {
```

Comment:
Commit message nit: "...depend**s** on glibc ... a**n** Alpine base image that comes with gli**b**c"

Reply:
Done.